### ISSUE TITLE
Partners and Sponsors

### ISSUE THREAD

**Issue by adrisua** [hackseq/October_2016_private#3]
Thanks for connecting with Zaira @ababaian. She was a bit confused so for future communications to potential partners please check with me to make sure we have not connected with them already +1:

**Comment by ababaian** [comment]
I emailed her before the meeting/assigned roles.

Do we have a list of contacts / people we've reached out to?

**Comment by adrisua** [comment]
Yes October_2016_private/Partners-Sponsors/Partners_Sponsors.csv
and October_2016_private/Team-Leaders/Team_Leaders_List.csv

**Comment by ababaian** [comment]
Sweet, can I add my Jobs in Science Podcast and GrasPods which can do communication and has a few people interested in volunteering for the event.

**Comment by adrisua** [comment]
Please go ahead.

**Comment by ababaian** [comment]
@adrisua can you please request high resolution logos from all sponsors / community partners. Some organizations it's harder to find the logos to use on the website and we'll need them for the event and bunch of places we'll put them.
